src/utils/httptransport.ts

```diff
@@ -1,80 +1,84 @@
-const METHODS = {
-		GET: 'GET',
-		POST: 'POST',
-		PUT: 'PUT',
-		DELETE: 'DELETE',
+enum METHODS {
+		GET = 'GET',
+		POST = 'POST',
+		PUT = 'PUT',
+		DELETE = 'DELETE',
 };
 
-// Самая простая версия. Реализовать штучку со всеми проверками им предстоит в конце спринта
-// Необязательный метод
-function queryStringify(data) {
+function queryStringify(data:Record<string, any>) {
   if (typeof data !== 'object') {
 			throw new Error('Data must be object');
 	}
   
-	// Здесь достаточно и [object Object] для объекта
   const keys = Object.keys(data);
   return keys.reduce((result, key, index) => {
     return `${result}${key}=${data[key]}${index < keys.length - 1 ? '&' : ''}`;
   }, '?');
 }
+interface Options {
+	timeout: number,
+	headers: Record<string, string>,
+	method: string,
+	data: Record<string, any> | XMLHttpRequestBodyInit
+}
+
+export class HTTPTransport {
+	get = (url:string, options: Options) => {
```

Comment:
Метод здесь уже не нужно пробрасывать, можно использовать дженерик Omit
```suggestion
	get = (url:string, options: Omit<Options, 'method'>) => {
```